.circleci/config.yml

```
@@ -25,6 +25,17 @@ jobs:
       - checkout
       - run: make build
 
+  check-dependencies:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - run:
+          name: Check unused Go dependencies
+          command: |
+            go mod tidy
```

Using `-v` will help catching odd situations like a broken connection to the internet.
```suggestion
            go mod tidy -v
```

---

I was thinking about including it but didn’t now that so perfect :)